ouroboros-consensus-byron/test/Test/ThreadNet/DualPBFT.hs

```diff
@@ -274,7 +274,7 @@ genDualPBFTTestConfig numSlots params = do
 
     return TestConfig {
           nodeRestarts = noRestarts
-        , slotLengths  = singletonSlotLengths (slotLengthFromSec 20)
```

Comment:
Test config doesn't allow for multiple slot lengths, because we can only change slot lengths at hard fork boundaries.